public/app/plugins/datasource/elasticsearch/components/QueryEditor/AnnotationQueryEditor.tsx

```diff
@@ -17,9 +18,10 @@ export function ElasticsearchAnnotationsQueryEditor(props: Props) {
   const annotation = props.annotation!;
   const onAnnotationChange = props.onAnnotationChange!;
 
+  const styles = useStyles2(getStyles);
   return (
     <>
-      <div className="gf-form-group">
+      <div className={styles.container}>
```

Comment:
Instead of adding these styles, you can try using the Stack component with the `gap` prop.

Reply:
do I use yarn install to install stack ?

Reply:
It's a component from [grafana/ui](https://developers.grafana.com/ui/canary/index.html?path=/story/general-layout-stack--align-items-examples).

Reply:
Stack component from material UI? that will also require material UI installation right? lmk

Reply:
You can add an empty div as the last child of Stack (with a comment explaining why it's there) to preserve the spacing.

Reply:
![image](https://github.com/grafana/grafana/assets/57278642/560c62bb-5351-4778-ae9b-f48d1615256c)
works now!